src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/UWP/PopupRenderer.uwp.cs

```diff
@@ -283,6 +300,7 @@ protected virtual void Dispose(bool disposing)
 				Control = null;
 
 				Closed -= OnClosing;
```

Comment:
```suggestion
				Closing -= OnClosing;
```

Reply:
`Closing` is an object from the UWP `Flyout` and can't be renamed. I can take the spirit of this change and rename `OnClosing -> OnClosed`
```c#
Closed -= OnClosed
```

Reply:
This is a great catch as we would have a potential memory leak here.